src/Ombi/wwwroot/translations/de.json

```diff
@@ -82,7 +82,8 @@
       "TopRatedMovies": "Am besten bewertete Filme",
       "NowPlayingMovies": "Im Kino",
       "HomePage": "Startseite",
-      "Trailer": "Trailer"
+      "Trailer": "Trailer",
+      "PlaceholderText": "search for a movie here..."
```

Comment:
This text has now been duplicated a lot! That means there will be more than 1 potential translation for it, Maybe if we movie this and the other duplicates under the `"Common"` section

Reply:
they aren't technically duplicates. one is the placeholder for tv/movie and the other is for tv/movie _requests_ 

```
"PlaceholderText": "search for a movie here..."
"PlaceholderText": "search for an existing movie request here..."

"PlaceholderText": "search for an existing tv show request here..."
"PlaceholderText": "search for a tv show here..."

```